src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/IAzureDevOpsClient.cs

```diff
@@ -0,0 +1,126 @@
+        /// <param name="project">Project that the feed was created in</param>
+        /// <param name="feedIdentifier">Name or id of the feed</param>
+        /// <returns></returns>
+        Task DeleteFeedAsync(string accountName, string project, string feedIdentifier);
+
+        /// <summary>
+        /// Deletes a NuGet package version from a feed.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="project">Project that the feed was created in</param>
+        /// <param name="feedIdentifier">Name or id of the feed</param>
+        /// <param name="packageName">Name of the package</param>
+        /// <param name="version">Version to delete</param>
+        /// <returns></returns>
+        Task DeleteNuGetPackageVersionFromFeedAsync(string accountName, string project, string feedIdentifier, string packageName, string version);
+
+        /// <summary>
+        ///     Fetches an specific AzDO build based on its ID.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="projectName">Project name</param>
+        /// <param name="buildId">Id of the build to be retrieved</param>
+        /// <returns>AzureDevOpsBuild</returns>
+        Task<AzureDevOpsBuild> GetBuildAsync(string accountName, string projectName, long buildId);
+
+        /// <summary>
+        /// Gets a specified Artifact feed with their pacckages in an Azure DevOps account.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name.</param>
+        /// <param name="feedIdentifier">ID or name of the feed.</param>
+        /// <returns>List of Azure DevOps feeds in the account.</returns>
+        Task<AzureDevOpsFeed> GetFeedAndPackagesAsync(string accountName, string project, string feedIdentifier);
+
+        /// <summary>
+        /// Gets a specified Artifact feed in an Azure DevOps account.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="feedIdentifier">ID or name of the feed</param>
+        /// <returns>List of Azure DevOps feeds in the account</returns>
+        Task<AzureDevOpsFeed> GetFeedAsync(string accountName, string project, string feedIdentifier);
+
+        /// <summary>
+        /// Gets all Artifact feeds along with their packages in an Azure DevOps account.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name.</param>
+        /// <returns>List of Azure DevOps feeds in the account.</returns>
+        Task<List<AzureDevOpsFeed>> GetFeedsAndPackagesAsync(string accountName);
+
+        /// <summary>
+        /// Gets a specified Artifact feed in an Azure DevOps account.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="feedIdentifier">ID or name of the feed</param>
```

Comment:
NIT: too much documentation here. There is no feedIdentifier param

Reply:
This is actually the documentation for the wrong method haha. I'll fix all these docs.

Reply:
I think I've updated all the incorrect methods in both the Interface and implementation docs.